lib/krikri/harvester.rb

```diff
@@ -102,7 +102,12 @@ def get_record(_)
     # @return [Boolean]
     def run(activity_uri = nil)
       log :info, 'harvest is running'
-      records.each { |rec| rec.save(activity_uri) }
+      begin
```

Comment:
I think that sounds reasonable. I'd love to talk through more how errors should get handled by harvesters and when we want to fail the whole thing.